src/main/java/org/opensearch/security/configuration/DlsFlsValveImpl.java

```diff
@@ -117,6 +117,11 @@ public boolean invoke(final ActionRequest request, final ActionListener<?> liste
                 listener.onFailure(new OpenSearchSecurityException("Resize is not supported when FLS or DLS or Fieldmasking is activated"));
                 return false;
             }
+
+            if(action.contains("plugins/replication")) {
```

Comment:
What are the actual action names? Where do they originate?

Reply:
these action names triggered from replication plugin